src/components/AssetManagementPanel/AssetManagementForm/FormVariants/RejectSurrenderedForm/RejectSurrenderedForm.tsx

```diff
@@ -36,14 +36,15 @@ export const RejectSurrenderedForm: FunctionComponent<RejectSurrenderedFormProps
   restoreTokenState,
 }) => {
   const { showOverlay } = useContext(OverlayContext);
-  const { endorsementChain, pending } = useEndorsementChain(tokenRegistryAddress, tokenId);
+  let lastBeneficiary = "Loading...";
```

Comment:
doesnt make sense; change back to how it was coded

can extract the "Loading..." text into a shared variable if u want to dry things